### PR TITLE
Bugfix/1

### PR DESCRIPTION
**Pull request by aarbrown**
Issue #1 with test and fix

### CODE REVIEW

**Comment by aarbrown** [comment]
I missed the git submodules when I tested locally.  I'm working on getting the tests to pass locally and will update the PR

File: tests/_data/container.php [hunk 1]
```diff
@@ -85,6 +85,15 @@ function (ServerRequestInterface $request, ResponseInterface $response) {
             }
         );
 
+        $app->post(
```
Comment (herloct) [inline tests/_data/container.php:88]: I think like we could use existing `POST /rest` for this.
No need to create new route.

File: tests/functional/SeeParsedBodyIsNotNullCept.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?php
```
Comment (herloct) [inline tests/functional/SeeParsedBodyIsNotNullCept.php:1]: By using existing `POST /rest`, we could adjust the acception test to something like:

``` php
<?php
$I = new FunctionalTester($scenario);

$data = ['testing' => '123'];
$I->wantTo('test that the parsed request body is returned in tact.');
$I->haveHttpHeader('content-type', 'application/json');
$I->sendPOST('/rest', $data);
$I->seeResponseCodeIs(\Codeception\Util\HttpCode::OK);
$I->seeResponseContainsJson([
    'rawBody' => json_encode($data),
    'formParams' => $data
]);
```

**Review by herloct — CHANGES_REQUESTED** [review]
Would you mind apply my suggestions?

**Review by herloct — APPROVED** [review]

**Pull request merged**